### PR TITLE
Updating manager url to be place holder

### PR DESCRIPTION
**Pull request by Vbhatia20**
We shouldn't be providing manager url options in yaml as customer can be on prod1 or prod2 or prod3

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ env:
 - name: ACCOUNT_ID
   value: <account id>
 - name: MANAGER_HOST_AND_PORT
-  value: <https://app.harness.io OR https://app.harness.io/gratis>
+  value: <manager url>
```
Comment (kat-enos) [inline docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md:104]:
```suggestion
  value: <manager-url>
```

File: docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ env:
 - name: CLIENT_TOOLS_DOWNLOAD_DISABLED
   value: "true"
 - name: LOG_STREAMING_SERVICE_URL  
-  value: "https://app.harness.io/log-service/ OR https://app.harness.io/gratis/log-service/"
+  value: "<manager url>/log-service/"
```
Comment (kat-enos) [inline docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md:130]:
```suggestion
  value: "<manager-url>/log-service/"
```

File: docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md (hunk 1 above)
Comment (Vbhatia20) [reply docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md]: done. Thankyou.

File: docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md (hunk 2 above)
Comment (Vbhatia20) [reply docs/platform/2_Delegates/delegate-reference/YAML/example-kubernetes-manifest-harness-delegate.md]: done. Thankyou.

**Review by kat-enos — APPROVED** [review]

**Comment by kat-enos** [comment]
Sounds like you want me to merge this @Vbhatia20 , so I'll go ahead and do that. Cheers.

**Pull request merged**